bayesian/api_v1.py

```diff
@@ -462,23 +463,25 @@ def post():
             raise HTTPError(400, error="Expected JSON request")
 
         if 'manual_tagging' not in input_json:
-            if 'component' not in input_json:
-                raise HTTPError(400, error="Expected component name in the request")
+            if 'ecosystem' not in input_json:
+                raise HTTPError(400, error="Expected ecosystem in the request")
 
-            if 'intent' not in input_json:
-                raise HTTPError(400, error="Expected intent in the request")
+            if 'data' not in input_json:
+                raise HTTPError(400, error="Expected data in the request")
 
+            init_selinon()
             s3 = StoragePool.get_connected_storage('S3UserIntent')
 
             # Store data
-            return s3.store_in_bucket(input_json)
+            return s3.store_master_tags(input_json)
         else:
             if 'user' not in input_json:
                 raise HTTPError(400, error="Expected user name in the request")
 
             if 'data' not in input_json:
                 raise HTTPError(400, error="Expected tags in the request")
 
+            init_selinon()
```

Comment:
It would be nice to have a decorator for this so this is not spread across sources randomly.

Reply:
(but it is not requirement from my side)